File: src/com/jatools/vo/calc/MaterAttrUpdate.java
package com.jatools.vo.calc;

import com.jatools.common.Global;
import com.jatools.common.constant.GlobalConstant;
import com.jatools.common.intf.ReviewBill;

public class MaterAttrUpdate implements ReviewBill{
	private String billid;//				
	private String billno;//	//		
	private String orgid;//	//		
	private String status;//	//		
	private String rebate;//	//		
	private String newrebate;//	//		
	private String rebatetype;//	//		
	private String newrebatetype;//	//		
	private String tagtype;//	//		
	private String newtagtype;//	//		
	private String stringworkprice;//	//		
	private String newstringworkprice;//	//		
	private String specialworkprice;//	//		
	private String newspecialworkprice;//	//		
	private String ornastate;//	//		
	private String newornastate;//	//		
	private String isdbllabel;	//		
	private String newisdbllabel;	//		
	private String mainshapeid;//	//	
	private String newmainshapeid;//	//	
	private String colorid;//	//	 
	private String newcolorid;//	//	 
	private String cleanid;//	//	
	private String newcleanid;//	//	
	private String maincolorgradeid;//	//	
	private String newmaincolorgradeid;//	//	
	private String cutid;//	//	
	private String newcutid;//	//	
	private String cutwidescale;//	//		
	private String newcutwidescale;//	//		
	private String cutdeepscale;//	//		
	private String newcutdeepscale;//	//		
	private String symmetryid;//	//	
	private String newsymmetryid;//	//	
	private String polishineid;//	//	
	private String newpolishineid;//	//	
	private String fluorescenceid;//	//	
	private String newfluorescenceid;//	//	
	private String waistlineid;//	//	
	private String newwaistlineid;//	//	
	private String vertexid;//	//	
	private String newvertexid;//	//	
	private String bracketcolorid;//	//	
	private String newbracketcolorid;//	//	
	private String wearid;//	//		
	private String newwearid;//	//		
	private String styletypeid;//	//		
	private String newstyletypeid;//	//		
	private String ismutipart;	//		 0  1
	private String newismutipart;	//		 0  1
	private String partcontent;//	//		
	private String newpartcontent;//	//		
	private String zodiac;//	//		
	private String newzodiac;//	//		
	private String createdate;//	//		
	private String createid;//	//		
	private String updatedate;//	//		
	private String updateid;//	//		
	private String memo;//	//		
	private String updatereason;//	//		
	private String rebateamount;//	//		
	private String newrebateamount;//	//	
	private String ornacode;
	private String ornabarcode;
	
	
	public String getOrnabarcode() {
		return ornabarcode;
	}
	public void setOrnabarcode(String ornabarcode) {
		this.ornabarcode = ornabarcode;
	}
	public String getOrnacode() {
		return ornacode;
	}
	public void setOrnacode(String ornacode) {
		this.ornacode = ornacode;
	}
	public String getBillid() {
		return billid;
	}
	public void setBillid(String billid) {
		this.billid = billid;
	}
	public String getBillno() {
		return billno;
	}
	public void setBillno(String billno) {
		this.billno = billno;
	}
	public String getOrgid() {
		return orgid;
	}
	public void setOrgid(String orgid) {
		this.orgid = orgid;
	}
	public String getStatus() {
		return status;
	}
	public void setStatus(String status) {
		this.status = status;
	}
	public String getRebate() {
		return rebate;
	}
	public void setRebate(String rebate) {
		this.rebate = rebate;
	}
	public String getNewrebate() {
		return newrebate;
	}
	public void setNewrebate(String newrebate) {
		this.newrebate = newrebate;
	}
	public String getRebatetype() {
		return rebatetype;
	}
	public void setRebatetype(String rebatetype) {
		this.rebatetype = rebatetype;
	}
	public String getNewrebatetype() {
		return newrebatetype;
	}
	public void setNewrebatetype(String newrebatetype) {
		this.newrebatetype = newrebatetype;
	}
	public String getTagtype() {
		return tagtype;
	}
	public void setTagtype(String tagtype) {
		this.tagtype = tagtype;
	}
	public String getNewtagtype() {
		return newtagtype;
	}
	public void setNewtagtype(String newtagtype) {
		this.newtagtype = newtagtype;
	}
	public String getStringworkprice() {
		return stringworkprice;
	}
	public void setStringworkprice(String stringworkprice) {
		this.stringworkprice = stringworkprice;
	}
	public String getNewstringworkprice() {
		return newstringworkprice;
	}
	public void setNewstringworkprice(String newstringworkprice) {
		this.newstringworkprice = newstringworkprice;
	}
	public String getSpecialworkprice() {
		return specialworkprice;
	}
	public void setSpecialworkprice(String specialworkprice) {
		this.specialworkprice = specialworkprice;
	}
	public String getNewspecialworkprice() {
		return newspecialworkprice;
	}
	public void setNewspecialworkprice(String newspecialworkprice) {
		this.newspecialworkprice = newspecialworkprice;
	}
	public String getOrnastate() {
		return ornastate;
	}
	public void setOrnastate(String ornastate) {
		this.ornastate = ornastate;
	}
	public String getNewornastate() {
		return newornastate;
	}
	public void setNewornastate(String newornastate) {
		this.newornastate = newornastate;
	}
	public String getIsdbllabel() {
		return isdbllabel;
	}
	public void setIsdbllabel(String isdbllabel) {
		this.isdbllabel = isdbllabel;
	}
	public String getNewisdbllabel() {
		return newisdbllabel;
	}
	public void setNewisdbllabel(String newisdbllabel) {
		this.newisdbllabel = newisdbllabel;
	}
	public String getMainshapeid() {
		return mainshapeid;
	}
	public void setMainshapeid(String mainshapeid) {
		this.mainshapeid = mainshapeid;
	}
	public String getNewmainshapeid() {
		return newmainshapeid;
	}
	public void setNewmainshapeid(String newmainshapeid) {
		this.newmainshapeid = newmainshapeid;
	}
	public String getColorid() {
		return colorid;
	}
	public void setColorid(String colorid) {
		this.colorid = colorid;
	}
	public String getNewcolorid() {
		return newcolorid;
	}
	public void setNewcolorid(String newcolorid) {
		this.newcolorid = newcolorid;
	}
	public String getCleanid() {
		return cleanid;
	}
	public void setCleanid(String cleanid) {
		this.cleanid = cleanid;
	}
	public String getNewcleanid() {
		return newcleanid;
	}
	public void setNewcleanid(String newcleanid) {
		this.newcleanid = newcleanid;
	}
	public String getMaincolorgradeid() {
		return maincolorgradeid;
	}
	public void setMaincolorgradeid(String maincolorgradeid) {
		this.maincolorgradeid = maincolorgradeid;
	}
	public String getNewmaincolorgradeid() {
		return newmaincolorgradeid;
	}
	public void setNewmaincolorgradeid(String newmaincolorgradeid) {
		this.newmaincolorgradeid = newmaincolorgradeid;
	}
	public String getCutid() {
		return cutid;
	}
	public void setCutid(String cutid) {
		this.cutid = cutid;
	}
	public String getNewcutid() {
		return newcutid;
	}
	public void setNewcutid(String newcutid) {
		this.newcutid = newcutid;
	}
	public String getCutwidescale() {
		return cutwidescale;
	}
	public void setCutwidescale(String cutwidescale) {
		this.cutwidescale = cutwidescale;
	}
	public String getNewcutwidescale() {
		return newcutwidescale;
	}
	public void setNewcutwidescale(String newcutwidescale) {
		this.newcutwidescale = newcutwidescale;
	}
	public String getCutdeepscale() {
		return cutdeepscale;
	}
	public void setCutdeepscale(String cutdeepscale) {
		this.cutdeepscale = cutdeepscale;
	}
	public String getNewcutdeepscale() {
		return newcutdeepscale;
	}
	public void setNewcutdeepscale(String newcutdeepscale) {
		this.newcutdeepscale = newcutdeepscale;
	}
	public String getSymmetryid() {
		return symmetryid;
	}
	public void setSymmetryid(String symmetryid) {
		this.symmetryid = symmetryid;
	}
	public String getNewsymmetryid() {
		return newsymmetryid;
	}
	public void setNewsymmetryid(String newsymmetryid) {
		this.newsymmetryid = newsymmetryid;
	}
	public String getPolishineid() {
		return polishineid;
	}
	public void setPolishineid(String polishineid) {
		this.polishineid = polishineid;
	}
	public String getNewpolishineid() {
		return newpolishineid;
	}
	public void setNewpolishineid(String newpolishineid) {
		this.newpolishineid = newpolishineid;
	}
	public String getFluorescenceid() {
		return fluorescenceid;
	}
	public void setFluorescenceid(String fluorescenceid) {
		this.fluorescenceid = fluorescenceid;
	}
	public String getNewfluorescenceid() {
		return newfluorescenceid;
	}
	public void setNewfluorescenceid(String newfluorescenceid) {
		this.newfluorescenceid = newfluorescenceid;
	}
	public String getWaistlineid() {
		return waistlineid;
	}
	public void setWaistlineid(String waistlineid) {
		this.waistlineid = waistlineid;
	}
	public String getNewwaistlineid() {
		return newwaistlineid;
	}
	public void setNewwaistlineid(String newwaistlineid) {
		this.newwaistlineid = newwaistlineid;
	}
	public String getVertexid() {
		return vertexid;
	}
	public void setVertexid(String vertexid) {
		this.vertexid = vertexid;
	}
	public String getNewvertexid() {
		return newvertexid;
	}
	public void setNewvertexid(String newvertexid) {
		this.newvertexid = newvertexid;
	}
	public String getBracketcolorid() {
		return bracketcolorid;
	}
	public void setBracketcolorid(String bracketcolorid) {
		this.bracketcolorid = bracketcolorid;
	}
	public String getNewbracketcolorid() {
		return newbracketcolorid;
	}
	public void setNewbracketcolorid(String newbracketcolorid) {
		this.newbracketcolorid = newbracketcolorid;
	}
	public String getWearid() {
		return wearid;
	}
	public void setWearid(String wearid) {
		this.wearid = wearid;
	}
	public String getNewwearid() {
		return newwearid;
	}
	public void setNewwearid(String newwearid) {
		this.newwearid = newwearid;
	}
	public String getStyletypeid() {
		return styletypeid;
	}
	public void setStyletypeid(String styletypeid) {
		this.styletypeid = styletypeid;
	}
	public String getNewstyletypeid() {
		return newstyletypeid;
	}
	public void setNewstyletypeid(String newstyletypeid) {
		this.newstyletypeid = newstyletypeid;
	}
	public String getIsmutipart() {
		return ismutipart;
	}
	public void setIsmutipart(String ismutipart) {
		this.ismutipart = ismutipart;
	}
	public String getNewismutipart() {
		return newismutipart;
	}
	public void setNewismutipart(String newismutipart) {
		this.newismutipart = newismutipart;
	}
	public String getPartcontent() {
		return partcontent;
	}
	public void setPartcontent(String partcontent) {
		this.partcontent = partcontent;
	}
	public String getNewpartcontent() {
		return newpartcontent;
	}
	public void setNewpartcontent(String newpartcontent) {
		this.newpartcontent = newpartcontent;
	}
	public String getZodiac() {
		return zodiac;
	}
	public void setZodiac(String zodiac) {
		this.zodiac = zodiac;
	}
	public String getNewzodiac() {
		return newzodiac;
	}
	public void setNewzodiac(String newzodiac) {
		this.newzodiac = newzodiac;
	}
	public String getCreatedate() {
		return createdate;
	}
	public void setCreatedate(String createdate) {
		this.createdate = createdate;
	}
	public String getCreateid() {
		return createid;
	}
	public void setCreateid(String createid) {
		this.createid = createid;
	}
	public String getUpdatedate() {
		return updatedate;
	}
	public void setUpdatedate(String updatedate) {
		this.updatedate = updatedate;
	}
	public String getUpdateid() {
		return updateid;
	}
	public void setUpdateid(String updateid) {
		this.updateid = updateid;
	}
	public String getMemo() {
		return memo;
	}
	public void setMemo(String memo) {
		this.memo = memo;
	}
	public String getUpdatereason() {
		return updatereason;
	}
	public void setUpdatereason(String updatereason) {
		this.updatereason = updatereason;
	}
	public String getRebateamount() {
		return rebateamount;
	}
	public void setRebateamount(String rebateamount) {
		this.rebateamount = rebateamount;
	}
	public String getNewrebateamount() {
		return newrebateamount;
	}
	public void setNewrebateamount(String newrebateamount) {
		this.newrebateamount = newrebateamount;
	}
	@Override
	public String getBillCode() {
		return GlobalConstant.BILL_CODE_MATERATTRUPDATE;
	}
	@Override
	public String getPageUrl() {
		return Global.CONTEXT  + "/calc/materAttrUpdate.vm?user_action=toEditMaterAttrUpdate&id=" + this.billid;
	}
	@Override
	public String getBeanName() {
		return "materAttrUpdateDao";
	}
	@Override
	public String getOrgId() {
		return this.orgid;
	}
	
	
}

File: WebContent/js/calc/materAttrUpdate.js
/**
 * 初始化数据
 */
function setValues(){
	
}
function setPageContent(){
	var status = jQuery("#status").val();
	if(status!="1"){
		jQuery("#btnSave").attr("disabled", "disabled");
		jQuery("#btnJ").attr("disabled", "disabled");
	}

	loadUpdateMaterPeam("newrebate",jQuery("#rebate").val());	
	loadUpdateMaterPeam("newrebatetype",jQuery("#rebatetype").val());	
	loadUpdateMaterPeam("newrebateamount",jQuery("#rebateamount").val());	
	loadUpdateMaterPeam("newtagtype",jQuery("#tagtype").val());
	loadUpdateMaterPeam("newstringworkprice",jQuery("#newstringworkprice").val());	
	loadUpdateMaterPeam("newspecialworkprice",jQuery("#specialworkprice").val());	
	loadUpdateMaterPeam("newcutwidescale",jQuery("#cutwidescale").val());	
	loadUpdateMaterPeam("newcutdeepscale",jQuery("#cutdeepscale").val());	
	loadUpdateMaterPeam("newpartcontent",jQuery("#partcontent").val());	
	loadUpdateMaterPeam("newornastate",jQuery("#ornastate").val());
}
/**
 * 查找饰品
 * @param type
 */
function loadCode(type){
	var code="";
	if(type=="1")
		code = jQuery("#ornacode").val();
	else
		code = jQuery("#ornabarcode").val();

	showLayer(true);
	MaterAttrUpdateDwr.getMaterCode(code,type,function(mater){
		showLayer(false);
		if(mater==null){
			jQuery("#ornabarcode").val("");
			jQuery("#ornacode").val("");
			alert("不存在此饰品");
		}else
			loadMater(mater);
	});
}
/**
 * 修改时加载属性各个段
 * @param peam
 * @param value
 */
function loadUpdateMaterPeam(peam,value){
//	if(null!=value&&""!=value)
//		jQuery("#"+peam+"").removeAttr("disabled");
//	else
//		jQuery("#"+peam+"").attr("disabled", "disabled");
}
/**
 * 加载饰品属性各个段
 * @param peam
 */
function loadMaterPeam(peam1,peam2,value){
	jQuery("#"+peam1+"").val("");
	jQuery("#"+peam2+"").val("");
	if(null!=value){
		jQuery("#"+peam1+"").val(value);
		//jQuery("#"+peam2+"").removeAttr("disabled");
	}else{
		//jQuery("#"+peam2+"").attr("disabled", "disabled");
	}
}
/**
 * 加载属性验证
 * @param peam
 * @param value
 */
function loadDataTypePeam(peam,value){
	if(null!=value&&""!=value){
		jQuery("#"+peam+"").attr("dataType", "Double");
	}else{
		jQuery("#"+peam+"").attr("dataType", "");
	}
	
}
/**
 * 加载饰品属性
 * @param mater
 */
function loadMater(mater){
	jQuery("#ornabarcode").val(mater.ornaBarcode);
	jQuery("#ornacode").val(mater.ornaCode);
	
	loadMaterPeam("rebate","newrebate",mater.rebate);

	loadMaterPeam("rebatetype","newrebatetype",mater.rebateType);
	
	loadMaterPeam("rebateamount","newrebateamount",mater.rebateAmount);

	loadMaterPeam("tagtype","newtagtype",mater.tagType);		

	loadMaterPeam("stringworkprice","newstringworkprice",mater.stringWorkPrice);
	
	loadMaterPeam("specialworkprice","newspecialworkprice",mater.specialWorkPrice);

	loadMaterPeam("mainshapeid","newmainshapeid",mater.mainShapeId);

	loadMaterPeam("colorid","newcolorid",mater.colorId);
	
	loadMaterPeam("cleanid","newcleanid",mater.cleanId);

	loadMaterPeam("maincolorgradeid","newmaincolorgradeid",mater.mainColorGradeId);
	
	loadMaterPeam("cutid","newcutid",mater.cutId);

	loadMaterPeam("cutwidescale","newcutwidescale",mater.cutWideScale);
	
	loadMaterPeam("cutdeepscale","newcutdeepscale",mater.cutDeepScale);
	
	loadMaterPeam("symmetryid","newsymmetryid",mater.symmetryId);
	
	loadMaterPeam("polishineid","newpolishineid",mater.polishineId);
	
	loadMaterPeam("fluorescenceid","newfluorescenceid",mater.fluorescenceId);	
	
	loadMaterPeam("waistlineid","newwaistlineid",mater.waistlineId);

	loadMaterPeam("vertexid","newvertexid",mater.vertexId);	

	loadMaterPeam("bracketcolorid","newbracketcolorid",mater.bracketcolorId);

	loadMaterPeam("wearid","newwearid",mater.wearId);

	loadMaterPeam("styletypeid","newstyletypeid",mater.styleTypeId);

	loadMaterPeam("partcontent","newpartcontent",mater.partContent);
	
	loadMaterPeam("zodiac","newzodiac",mater.zodiac);
	
	loadMaterPeam("ornastate","newornastate",mater.state);
	
	
	jQuery("#isdbllabel")[0].checked= mater.isDblLabel=="1"?true:false;
	jQuery("#newisdbllabel")[0].checked= mater.isDblLabel=="1"?true:false;
	jQuery("#ismutipart")[0].checked= mater.isMutiPart=="1"?true:false;
	jQuery("#newismutipart")[0].checked= mater.isMutiPart=="1"?true:false;
}

function getBill(status){
	var headObj = {
			billid               :returnNull(jQuery("#billid").val()),//				                  
			billno               :returnNull(jQuery("#billno").val()),//	//		              
			orgid                :returnNull(jQuery("#orgid").val()),//	//		              
			status               :status,//	//		              
			rebate               :returnNull(jQuery("#rebate").val()),//	//		              
			newrebate            :returnNull(jQuery("#newrebate").val()),//	//		          
			rebatetype           :returnNull(jQuery("#rebatetype").val()),//	//		          
			newrebatetype        :returnNull(jQuery("#newrebatetype").val()),//	//		      
			tagtype              :returnNull(jQuery("#tagtype").val()),//	//		            
			newtagtype           :returnNull(jQuery("#newtagtype").val()),//	//		          
			stringworkprice      :returnNull(jQuery("#stringworkprice").val()),//	//		    
			newstringworkprice   :returnNull(jQuery("#newstringworkprice").val()),//	//		  
			specialworkprice     :returnNull(jQuery("#specialworkprice").val()),//	//		    
			newspecialworkprice  :returnNull(jQuery("#newspecialworkprice").val()),//	//		
			ornastate            :returnNull(jQuery("#ornastate").val()),//	//		          
			newornastate         :returnNull(jQuery("#newornastate").val()),//	//		        
			isdbllabel           :returnNull(jQuery("#isdbllabel")[0]==null?null:jQuery("#isdbllabel")[0].checked==false?'0':'1'),	//		 0  1		            
			newisdbllabel        :returnNull(jQuery("#newisdbllabel")[0]==null?null:jQuery("#newisdbllabel")[0].checked==false?'0':'1'),	//		 0  1		        
			mainshapeid          :returnNull(jQuery("#mainshapeid").val()),//	//	          
			newmainshapeid       :returnNull(jQuery("#newmainshapeid").val()),//	//	        
			colorid              :returnNull(jQuery("#colorid").val()),//	//	              
			newcolorid           :returnNull(jQuery("#newcolorid").val()),//	//	            
			cleanid              :returnNull(jQuery("#cleanid").val()),//	//	              
			newcleanid           :returnNull(jQuery("#newcleanid").val()),//	//	            
			maincolorgradeid     :returnNull(jQuery("#maincolorgradeid").val()),//	//	      
			newmaincolorgradeid  :returnNull(jQuery("#newmaincolorgradeid").val()),//	//	  
			cutid                :returnNull(jQuery("#cutid").val()),//	//	                
			newcutid             :returnNull(jQuery("#newcutid").val()),//	//	              
			cutwidescale         :returnNull(jQuery("#cutwidescale").val()),//	//		        
			newcutwidescale      :returnNull(jQuery("#newcutwidescale").val()),//	//		    
			cutdeepscale         :returnNull(jQuery("#cutdeepscale").val()),//	//		        
			newcutdeepscale      :returnNull(jQuery("#newcutdeepscale").val()),//	//		    
			symmetryid           :returnNull(jQuery("#symmetryid").val()),//	//	            
			newsymmetryid        :returnNull(jQuery("#newsymmetryid").val()),//	//	        
			polishineid          :returnNull(jQuery("#polishineid").val()),//	//	          
			newpolishineid       :returnNull(jQuery("#newpolishineid").val()),//	//	        
			fluorescenceid       :returnNull(jQuery("#fluorescenceid").val()),//	//	        
			newfluorescenceid    :returnNull(jQuery("#newfluorescenceid").val()),//	//	    
			waistlineid          :returnNull(jQuery("#waistlineid").val()),//	//	          
			newwaistlineid       :returnNull(jQuery("#newwaistlineid").val()),//	//	        
			vertexid             :returnNull(jQuery("#vertexid").val()),//	//	              
			newvertexid          :returnNull(jQuery("#newvertexid").val()),//	//	          
			bracketcolorid       :returnNull(jQuery("#bracketcolorid").val()),//	//	        
			newbracketcolorid    :returnNull(jQuery("#newbracketcolorid").val()),//	//	    
			wearid               :returnNull(jQuery("#wearid").val()),//	//		              
			newwearid            :returnNull(jQuery("#newwearid").val()),//	//		          
			styletypeid          :returnNull(jQuery("#styletypeid").val()),//	//		        
			newstyletypeid       :returnNull(jQuery("#newstyletypeid").val()),//	//		      
			ismutipart           :returnNull(jQuery("#ismutipart")[0]==null?null:jQuery("#ismutipart")[0].checked==false?'0':'1'),	//     
			newismutipart        :returnNull(jQuery("#newismutipart")[0]==null?null:jQuery("#newismutipart")[0].checked==false?'0':'1'),	//		 0  1   
			partcontent          :returnNull(jQuery("#partcontent").val()),//	//		        
			newpartcontent       :returnNull(jQuery("#newpartcontent").val()),//	//		      
			zodiac               :returnNull(jQuery("#zodiac").val()),//	//		              
			newzodiac            :returnNull(jQuery("#newzodiac").val()),//	//		          
			createdate           :returnNull(jQuery("#createdate").val()),//	//		          
			createid             :returnNull(jQuery("#createid").val()),//	//		            
			updatedate           :returnNull(jQuery("#updatedate").val()),//	//		          
			updateid             :returnNull(jQuery("#updateid").val()),//	//		            
			memo                 :returnNull(jQuery("#memo").val()),//	//		                
			updatereason         :returnNull(jQuery("#updatereason").val()),//	//		        
			rebateamount         :returnNull(jQuery("#rebateamount").val()),//	//		        
			newrebateamount      :returnNull(jQuery("#newrebateamount").val()),//	//	      
			ornacode             :returnNull(jQuery("#ornacode").val()),     
			ornabarcode          :returnNull(jQuery("#ornabarcode").val())                                        
	};
	return headObj;
}
function returnNull(v){
	if(v==""||v==null)
		return null;
	else
		return v;
}
function saveMaterAttrUpdate(status){
	if(Validator.Validate($("frm"),3)){
		confirm("确定保存?", function(){	
			showLayer(true);
    		MaterAttrUpdateDwr.saveOrUpdateMaterAttrUpdate(getBill(status), function(data){
    			showLayer(false);
    			var m = data.split("_");
    			if(m[0]=="2"){
    				alert(m[1]);
    			}else{
    				alert(m[1], function(){
    					window.location = "materAttrUpdate.vm";
    				});
    			}
    		});
    	});
	}
}

function checkNum(str){
	var value = jQuery("#"+str+"").val();
	if(value=="")
		return false;
	else if(Number(value)==0||value=="0"){
		jQuery("#"+str+"").val("");
		return false;
	}else if(Number(value)<0){
		jQuery("#"+str+"").val("");
		return false;
	}
	if(!isDecimal(value)){
		alert("请输入正确的数据");
		jQuery("#"+str+"").val("");
		return true;
	}return false;
}